charmander/resources/pizza_resources.py

```diff
@@ -31,13 +36,15 @@ def get_links(self, obj):
         links["self"] = Link.for_(
```

Comment:
Our resources have links. This is very useful for debugging as you can put links to useful things in the links for a resource and save frustrating fumbling during debugging.

Reply:
It's hard to have an interesting link right now just because the only model this service has is `pizza`. When I add the other models, there is a not about some non-trivial linking we add, for example from a `order` to it's child events or an `order` to it's child `pizzas`.